catalog/task/create-internal-request/0.1/create-internal-request.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-internal-request
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Submit a build request to add operator bundles to an index image
+  params:
+    - name: BINARY_IMAGE
+      type: string
+      description: FBC Image to be added to index image
+    - name: FBC_FRAGMENT
+      type: string
+      description: FBC fragment built by HACBS
+    - name: FROM_INDEX
+      type: string
+      description: Index image (catalog of catalogs) the FBC fragment will be added to    
+    - name: OVERWRITE_FROM_INDEX
+      type: string
+      description: Boolean indicating if the from_index should be overwritten      
+    - name: BUILD_TAGS      
+      type: string
+      description: List of additional tags the internal index image copy should be tagged with
+  steps:
+    - name: create-internal-request
+      image: quay.io/hacbs-release/release-utils:latest
```

Comment:
same comment about image

Reply:
If you are referring to your comment about the versioning in the old PR, I agree but  I would address this in a different ticket. If it is fine by you I will add the ticket to the backlog.

Reply:
Can't we just split the Dockerfile change to its own PR and merge that and not have to create more tickets? If we do it the way you recommend, this gets merged with latest, then we go to change it. When we change the image, is it task version 0.2? If so, 0.1 is not guaranteed to work forever. If not, why did we make a change to the files without creating a new version?

Reply:
done.